### DISCUSSION TITLE
Does SeisSol use m as the default unit?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#803] [Q&A] [answered]
I noticed that gmsh seems to have no units, does SeisSol use `m` as the default unit? Can it be modified to use `mm` as the unit？

**Comment by krenzland** [comment, marked as answer]
You can use any consistent unit system that you like.
See https://seissol.readthedocs.io/en/latest/scaling.html for more details
